build_helpers/markdownLoader.js

```diff
@@ -1,40 +1,42 @@
-"use strict";
+'use strict';
 
-var marked = require('marked');
+const { marked } = require('marked');
```

Comment:
marked needs to be imported slightly differently after the bump.
Every other change in this file is just prettier doing its thing.